qiskit/visualization/circuit_visualization.py

```
@@ -454,7 +460,13 @@ def _latex_circuit_drawer(
                     if filename.endswith(".pdf"):
                         os.rename(base + ".pdf", filename)
                     else:
-                        image.save(filename, "PNG")
+                        try:
+                            image.save(filename)
+                        except ValueError:
+                            logger.warning(
```

As above please raise an error instead of logging a warning